include/zephyr/bluetooth/audio/bap.h

```diff
@@ -41,6 +41,12 @@
 extern "C" {
 #endif
 
+#if defined(CONFIG_BT_BAP_BASS_MAX_SUBGROUPS)
+#define BT_BAP_BASS_MAX_SUBGROUPS CONFIG_BT_BAP_BASS_MAX_SUBGROUPS
+#else
+#define BT_BAP_BASS_MAX_SUBGROUPS 0
+#endif /* CONFIG_BT_BAP_BASS_MAX_SUBGROUPS*/
+
```

Comment:
```suggestion
#if defined(CONFIG_BT_BAP_BASS_MAX_SUBGROUPS)
# define BT_BAP_BASS_MAX_SUBGROUPS CONFIG_BT_BAP_BASS_MAX_SUBGROUPS
#else
# define BT_BAP_BASS_MAX_SUBGROUPS 0
#endif /* CONFIG_BT_BAP_BASS_MAX_SUBGROUPS*/

```

Reply:
Can you elaborate? Are you suggesting a space between `#` and `define`?

Reply:
Yes. It's ever readable.

Reply:
I think the space is very uncommon :)